README.md

```diff
@@ -1,8 +1,8 @@
-## Push Notification Plugin for Xamarin iOS and Android
+## Push Notification Plugin for Xamarin iOS and Android (Updated original package to use secure storage)
```

Comment:
We can remove (Updated original package to use secure storage)

Reply:
I'll adjust that.